pandas/plotting/_core.py

```diff
@@ -420,6 +420,12 @@ def hist_frame(
         matplotlib.axes.Axes or np.ndarray of them
             An ndarray is returned with one :class:`matplotlib.axes.Axes`
             per column when ``subplots=True``.
+
+        Notes
+        -----
+        Where pandas fails to infer the frequency information the automatic tick
+        resolution adjustment can be suppressed by the parameter:
+        `x_compact=True`  (e.g., `s.plot(x_compat=True)`).
```

Comment:
There's a typo here: `x_compact`.

Reply:
We have this in our user guide. @gauravchandok do you think it's valuable repeating this here?